src/components/calcite-slider/calcite-slider.scss

```diff
@@ -131,6 +126,24 @@ $tick-height: 4px;
 .thumb--minValue {
   transform: translate(-7px, -8px);
 }
+:host([debug]) {
+  .thumb {
+    .handle__label {
+      &.static {
+        background-color: var(--calcite-ui-blue-1);
+        opacity: 0.3;
+        color: transparent;
+      }
+      &.transformed:not([style*="translateX(0px)"]),
+      &.transformed:not([style*="margin-left: 0px"]),
+      &.transformed:not([style*="margin-right: 0px"]) {
+        background-color: red;
+        opacity: 0.3;
+        color: transparent;
+      }
+    }
+  }
+}
```

Comment:
Adding a `debug` attribute to any `calcite-slider` shows handy transparent color backgrounds on the static and moving parts helpful for debugging.

Reply:
I wonder if there is a way to tell stencil not to build that in dist builds

Reply:
Ah yes, good point, that would technically add to the bloat of the CSS.  I'll ask in the Stencil channel.